lms/models/lti_user.py

```diff
@@ -1,9 +1,10 @@
-from typing import NamedTuple
+from dataclasses import dataclass
 
 from lms.models.h_user import HUser
 
 
-class LTIUser(NamedTuple):
+@dataclass
```

Comment:
This being a dataclass doesn't seem like a huge improvement in this PR but it would make any further changes much easier.

Reply:
I think this is definitely better. I'm not sure if `NamedTuples` were picked because they were immutable, or whether they were picked for their `dataclass` like behaviours. I suspect the immutable aspect might be a retro-justification?

We certainly want to mutate them regularly (in tests at a minimum).